src/test/ui/repeat_count.stderr

```diff
@@ -39,22 +39,12 @@ error[E0308]: mismatched types
    |
 LL |     let f = [0; -4_isize];
    |                 ^^^^^^^^ expected `usize`, found `isize`
-   |
-help: you can convert an `isize` to `usize` and panic if the converted value wouldn't fit
-   |
-LL |     let f = [0; (-4_isize).try_into().unwrap()];
```

Comment:
This help message was wrong.

Reply:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=3e0d446869bd89792a113dbd8f140f15